cernopendata/modules/fixtures/data/docs/cms-releases-first-run2-data/cms-releases-first-run2-data.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "author": "CMS Collaboration",
+    "body": {
+      "content": "cms-releases-first-run2-data.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "experiment": "CMS"
+      },
+      {
+        "primary": "News"
+      }
+    ],
+    "created": "2021-12-20",
+    "experiment": "CMS",
+    "featured": 1,
```

Comment:
Thanks, I've added "featured" here and amended the old news to have the latest news box order correct.

Please see below a question about slug; I can rename the files accordingly depending on what's chosen as the final title and final slug.